### PR TITLE
bump gomodguard version

### PR DESCRIPTION
**Pull request by ryancurrah**
adds a new feature to allow blocking modules based on version constraints

### CODE REVIEW

File: pkg/config/config.go [hunk 1]
```diff
@@ -221,6 +221,10 @@ type LintersSettings struct {
 				Recommendations []string `mapstructure:"recommendations"`
 				Reason          string   `mapstructure:"reason"`
 			} `mapstructure:"modules"`
+			Versions []map[string]struct {
```
Comment (jirfag) [inline pkg/config/config.go:224]: please, add new params to docs - currently, it's `.golangci.example.yml`

Comment (jirfag) [reply pkg/config/config.go:224]: Also, could you please document somewhere (e.g. in description of a linter) what are differences from [depguard](https://github.com/OpenPeeDeeP/depguard) linter?

**Comment by jirfag** [comment]
I've disabled mac os tests: they are too slow and result in frequent timeouts.

File: pkg/config/config.go (hunk 1 above)
Comment (ryancurrah) [reply pkg/config/config.go:224]: Done, also added a reminder for contributors to this in the docs

Comment (jirfag) [reply pkg/config/config.go:224]: thank you!

**Comment by ryancurrah** [comment]
Resolves https://github.com/golangci/golangci-lint/issues/1087

**Pull request merged**